conda_kapsel/project_commands.py

```diff
@@ -28,6 +29,114 @@ def _is_windows():
     return (platform.system() == 'Windows')
 
 
+_ArgSpec = namedtuple('_ArgSpec', ['option', 'has_value'])
+
+_http_specs = (_ArgSpec('--kapsel-host', True), _ArgSpec('--kapsel-port', True), _ArgSpec('--kapsel-url-prefix', True),
+               _ArgSpec('--kapsel-no-browser', False))
```

Comment:
I used these namespaced command line options because I was afraid of stomping on existing options with our own that might have slightly different semantics or something. Since these options are intended mostly for use by scripts, it doesn't seem so bad. Also it helps explain why these options won't be in `--help`